Sources/Rownd/Models/Context/Context.swift

```diff
@@ -7,122 +7,18 @@
 
 import Foundation
 import ReSwift
-import ReSwiftThunk
 
-fileprivate let STORAGE_STATE_KEY = "RowndState"
-
-public struct RowndState: Codable, Hashable {
-    public var isInitialized = false
-    public var appConfig = AppConfigState()
-    public var auth = AuthState()
-    public var user = UserState()
-    public var passkeys = PasskeyState()
-    public var signIn = SignInState()
-}
-
-extension RowndState {
-    enum CodingKeys: String, CodingKey {
-        case appConfig, auth, user, signIn, passkeys
-    }
-
-    public init(from decoder: Decoder) throws {
-        let container = try decoder.container(keyedBy: CodingKeys.self)
-        appConfig = try container.decode(AppConfigState.self, forKey: .appConfig)
-        auth = try container.decode(AuthState.self, forKey: .auth)
-        user = try container.decode(UserState.self, forKey: .user)
-        passkeys = try container.decode(PasskeyState.self, forKey: .passkeys)
-        signIn = try container.decodeIfPresent(SignInState.self, forKey: .signIn) ?? SignInState()
-    }
-
-    static func save(state: RowndState) {
-        Task {
-            if let encoded = try? state.toJson() {
-                //            logger.trace("storing: \(encoded)")
-                Storage.store?.set(encoded, forKey: STORAGE_STATE_KEY)
-            }
-        }
-    }
+class Context {
```

Comment:
Why did you replace Store with Context class? Not a blocker! Just wondering what it's for.

Reply:
Two reasons:
1) Organization. `Context` seemed like the wrong name for a thing called `RowndState`.
2) Testing. Previously, it was difficult to unit test parts of the state code, because the SDK would init its own store and there was no way to really reset or isolate the state during testing. It's now possible to create a new "context" at will and then make the SDK use that fresh (or even reconfigured) context. This is likely not needed in "production," but makes it possible to test various scenarios as needed (which is happening in the new `StateTests` that I added).

Reply:
Okay make sense!